### PR TITLE
New drink syndicate screwdriver

### PR DESCRIPTION
**Pull request by Therandomhoboo**
# Document the changes in your pull request

A suggestion from @YashinHak about making a new Syndicate drink which is called the Syndicate Screwdriver, a combination of Syndicate bomb + Screwdriver to make a traitor based drink that gives you temp fast hands. 

It is complete, my first ever drink. Effects are subjected to change
Name: Syndicate Screwdriver
Boozepwr: 115
metabolization_rate = 1.5
Effect: Allows people part of the syndicate to do things faster and click a little faster (increased do_after speed and click_speed) by 0.75
Mix Message: "The mixture sparkles with a meancing red aura"
Taste description: "a tangy taste mixed with liquified Robustness"
Glass description: "A glass full of spite, haste and the need to greytide"

This makes the Syndicate Screwdriver the 3rd highest drink in the game. 1st being Bacchus (300), 2nd being Beepsky smash (120) and this one being the third (115)

Few edits have happened since this release to balance out what it gives. Metabolism rate is now at 0.75 (0.5 * 1.5 ) making it harder to use at lower quantities, and the output for the drink is now harder ( 5u of syndicate bomb and 5u of Screwdriver now give 1u of Syndicate Screwdriver) to stop easy spam. 

# Spriting
ScrewBomb
Sprite done by @zer0tactics

# Wiki Documentation

Syndicate Screwdriver - A special drink that invigorates members of the Syndicate to preform tasks & actions faster, however due to its high alcohol content it was determined to be too "unsafe" to be viable in missions.

# Changelog
:cl:  
rscadd: New drink, Syndicate Screwdriver  
imageadd: Syndicate Screwdriver sprite
/:cl:

### CODE REVIEW

**Comment by GDRdev** [comment]
fuck it add every tool as a drink

**Comment by Therandomhoboo** [comment]
> fuck it add every tool as a drink

Its to do with the two different drinks you dumb dum

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2676,3 +2674,27 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	quality = DRINK_GOOD
+	taste_description = "a tangy taste mixed with liquified Robustness"
+	glass_icon_state = "syndicate_screwdriver"
+	glass_name = "Syndicate Screwdriver"
+	glass_desc = "A glass full of spite, haste and the need to greytide"
+
+/datum/reagent/consumable/ethanol/syndicate_screwdriver/on_mob_metabolize(mob/living/carbon/human/M)
+	if(is_syndicate(M))
+		M.physiology.do_after_speed *= 0.75
+		M.next_move_modifier *= 0.75
```
Comment (Maia-J) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: make the 0.75 modifier a variable of the reagent

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (Therandomhoboo) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Done I think.

**Comment by azzzertyy** [comment]
not me stacking this with red-eye for insane clickspeed

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -14,8 +14,6 @@
 	taste_description = "alcohol"
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
 	var/boozepwr = 65 //Higher numbers equal higher hardness, higher hardness equals more intense alcohol poisoning
-	accelerant_quality = 5
```
Comment (Maia-J) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:17]: Forgot to fix this

**Comment by Therandomhoboo** [comment]
> not me stacking this with red-eye for insane clickspeed

good luck getting insanely hammered 
 
[Edit] Good luck, now it dont work

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 2 above)
Comment (Therandomhoboo) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:17]: done

**Review by cuackles — APPROVED** [review]

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 3]
```diff
@@ -932,3 +932,10 @@
 	results = list(/datum/reagent/consumable/ethanol/triumphal_arch = 10)
 	required_reagents = list(/datum/reagent/consumable/ethanol/mushi_kombucha = 5, /datum/reagent/consumable/ethanol/grappa = 2, /datum/reagent/consumable/lemonjuice = 2, /datum/reagent/gold = 1)
 	mix_message = "The mixture turns a deep golden hue."
+
+/datum/chemical_reaction/syndicate_screwdriver
+	name = "Syndicate Screwdriver"
+	id = /datum/reagent/consumable/ethanol/syndicate_screwdriver
+	results = list(/datum/reagent/consumable/ethanol/syndicate_screwdriver = 2)
```
Comment (adamsong) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]:
```suggestion
	results = list(/datum/reagent/consumable/ethanol/syndicate_screwdriver = 4)
```

**Review by adamsong — CHANGES_REQUESTED** [review]
Code mostly looks fine, balance is not my department

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm (hunk 3 above)
Comment (Therandomhoboo) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm]: So just as stated in Discord, there are other drinks/chems that do this (morphine, bastion & hearty punch) which is I assume to stop the drink being easily spammed. I don't mind changing it so it makes sense but I also think, I will tweak it so it'll have the same as Hearty punch in terms of required reagents. 
(5 Syndicate bomb + 5 Screwdriver = 1 syndicate screwdriver)

**Review by Majkl-J — APPROVED** [review]

**Comment by Maia-J** [comment]
My approve has no value but it looks fine codewise

**Comment by DeadlyBag** [comment]
The effects seem pretty high for something thats made so easily, imo would be cool if it used the new TC dust.
The modifiers themselves are super similar to the neural overclocker (0.7 from overclock vs 0.75 from this)
I think balance wise & thematically making it require TC dust would just be very cool

**Comment by Therandomhoboo** [comment]
> The effects seem pretty high for something thats made so easily, imo would be cool if it used the new TC dust. The modifiers themselves are super similar to the neural overclocker (0.7 from overclock vs 0.75 from this) I think balance wise & thematically making it require TC dust would just be very cool

So, whilst it is higher. The drawback from this is that currently: 

It has a low output (Getting only 1 part so 50 of syndibomb & screwdriver will only give 10u of Syndiscrewdriver), it metabolises quicker through your system meaning drinking only a swig or two will give you 1-2 seconds of use before wearing off.

Then finally the high boozepwr, means that you can't drink a full glass (even with alcohol tolerance) without pretty much handicapping yourself and combined that with the metabolic rate it means  you have to find the sweet spot between oding on alcohol or having no effect. With this, I think that is what balances the drink out. Only reason I don't want red dust is so its not exclusive to TC traitors, plus including a TC to make it will pretty much bait the drink as a free tester too (since only way to make the drink would be via reddust). I want the drink to give some sort of credible boost but a good slap to stop people abusing it and what I said covers that.

**Comment by Maia-J** [comment]
> The effects seem pretty high for something thats made so easily, imo would be cool if it used the new TC dust. The modifiers themselves are super similar to the neural overclocker (0.7 from overclock vs 0.75 from this) I think balance wise & thematically making it require TC dust would just be very cool

Why use tc dust when one can just get the 2TC sped up toolbox that lasts longer (As long as you can hide it from sec)
I am kinda fine on this, worst case one can fine tune it to have a negative effect or less of an effect

**Review by cuackles — APPROVED** [review]

**Review by ToasterBiome — APPROVED** [review]

**Review by adamsong — APPROVED** [review]
Not a fan of the ability to meta traitors, but the meta is hard to pull off, so I guess its fine

**Pull request merged**